### DISCUSSION TITLE
install without administrator

### DISCUSSION THREAD

**Discussion by gamer-sans987** [radubirsan/FlashBrowser#34] [Q&A]
hi, im in a windows 11 64-bits OS and i don't have any access to administrator, so i'm here to ask if it have any way to install in a local user?

**Comment by radubirsan** [comment]
we do not have this type of installation available.

**Reply by jrogers512** [reply]
If you make a [portable build](https://github.com/radubirsan/FlashBrowser/discussions/22), you will solve two problems with one solution.
